website/src/pages/use-cases/onboarding.tsx

```diff
@@ -0,0 +1,324 @@
+                    <CustomCarousel items={items} autoAdvance={true} hideCarouselNav={true} smallPanel={true} />
+                </div>
+            </div>
+        </ContentSection>
+
+        <div className="bg-gradient-onahau-fog py-8">
+            <ContentSection>
+                <div className="d-flex flex-column justify-content-center text-center px-lg-8">
+                    <BlockquoteWithLogo
+                        quote={`For our new developers, Sourcegraph has been invaluable to get to know the repository structure, to track down where code lives, 
+                            and self-service during their investigations.`}
+                        header="Convoy knows its codebase inside and out"
+                        by="Owen Kim, Senior Software Engineer at Convoy"
+                        logoHref="https://convoy.com"
+                        logoImage="/external-logos/convoy-logo.svg"
+                        linkText="Read the case study"
+                        link="/case-studies/convoy-improved-on-boarding"
+                    />
+                </div>
+            </ContentSection>
+        </div>
+
+        <div className="bg-light-gray2">
+            <ContentSection>
+                <div className="row d-flex flex-column mx-4 mx-lg-0 py-7 align-items-lg-center align-items-left">
+                    <div className="mb-5 d-flex flex-column px-lg-7 text-center">
+                        <h1 className="font-weight-bold">Give your team the onboarding experience they deserve.</h1>
+                        <p>
+                            Enable all your devs to find the answers they need to work more efficiently, ship code more
+                            confidently, and stay in flow.
+                        </p>
+                    </div>
+                    <div className="d-flex flex-column">
+                        <a
+                            className="btn btn-primary"
+                            href="https://info.sourcegraph.com/demo-request"
+                            title="Request a Demo."
+                        >
+                            Request a demo
+                        </a>
+                        <Link to="/use-cases" className="d-flex justify-content-center mt-4">
+                            <p className="font-weight-bold">Explore other use cases</p>
+                        </Link>
+                    </div>
+                </div>
+            </ContentSection>
+            <CustomerLogosSectionAnimated showButton={true} showSection={false} noCta={true} className="py-6" />
+        </div>
+
+        <ContentSection className="py-lg-7 py-5">
+            <div className="row d-flex">
+                <div className="col-lg-6">
+                    <h1 className="mb-5 font-weight-bold">Related resources</h1>
+                </div>
+                {blogListItems.map(item => (
+                    <BlogListItem key={item.title} blog={item} />
```

Comment:
Small nit on design, not the implementation – I see design's intention of matching the length of the paragraph to the title, but for design consistency, I like what we have implemented. +1 for leaving this as is @katjuell 🎉